### PR TITLE
More Search Fixes

### PR DESCRIPTION
**Pull request by jowlee**
### Summary
* Shrink search more btn on press
* Prevent open notification when search open
* Add tag search 
* Fix native -> webview transition w/ blank page routes on the webview (Fixes flashing)
* Add clicking search result to the history
* Fix styles
  * header font style on android
  * Click target of back and close in top bar of search
  * full width "Clear Search History"
  * Remove bottom bar under last search item in category
 
Merge after https://github.com/AudiusProject/audius-client/pull/401

### CODE REVIEW

File: src/components/search/TopBar.tsx [hunk 1]
```diff
@@ -12,61 +12,70 @@ import LottieView from 'lottie-react-native'
...
 import useSearchHistory from '../../store/search/hooks'
+import { usePushSearchRoute } from './utils'
 
 
 const IS_IOS = Platform.OS === 'ios'
 
 const styles = StyleSheet.create({
   topBar: {
-    height: IS_IOS ? 85 : 55,
+    height: IS_IOS ? 86 : 55,
```
Comment (raymondjacobson) [inline src/components/search/TopBar.tsx:31]: 86 or 87?

File: src/components/search/content/MoreContainer.tsx [hunk 2]
```diff
@@ -51,14 +51,50 @@ const MoreContainer = () => {
     pushWebRoute(getSearchRoute(searchResultQuery), 'search')
   }, [pushWebRoute, searchResultQuery])
 
+  const [shrinkAnim] = useState(new Animated.Value(1))
```
Comment (raymondjacobson) [inline src/components/search/content/MoreContainer.tsx:54]: just note to self that we should probs abstract this out of here at some point

File: src/components/search/content/SearchItem.tsx [hunk 3]
```diff
@@ -110,20 +108,22 @@ const UserSearchResult = ({ item: user }: UserSearchResultProps) => {
...
   const squareImageStyles = useTheme(styles.squareImage, { backgroundColor: 'neutralLight4' })
 
   const pushRoute = usePushSearchRoute()
+  const { appendSearchItem } = useSearchHistory()
   const onPress = useCallback(() => {
+    appendSearchItem(track.title)
     const trackRoute = getTrackRoute(track)
     pushRoute(trackRoute, 'search')
-  }, [track, pushRoute])
+  }, [track, pushRoute,appendSearchItem])
```
Comment (raymondjacobson) [inline src/components/search/content/SearchItem.tsx:123]: nit, should have a space here--how did this pass lint?

File: src/components/search/content/SearchItem.tsx [hunk 4]
```diff
@@ -137,20 +137,22 @@ const TrackSearchResult = ({ item: track }: TrackSearchResultProps) => {
...
+type PlaylistSearchResultProps = { isLast: boolean; item: SearchPlaylist }
+const PlaylistSearchResult = ({ isLast, item: playlist }: PlaylistSearchResultProps) => {
   const nameStyle = useTheme(styles.name, { color: 'neutral' })
   const userNameStyle = useTheme(styles.name, { color: 'neutralLight4' })
   const squareImageStyles = useTheme(styles.squareImage, { backgroundColor: 'neutralLight4' })
 
   const pushRoute = usePushSearchRoute()
+  const { appendSearchItem } = useSearchHistory()
   const onPress = useCallback(() => {
+    appendSearchItem(playlist.playlist_name)
```
Comment (raymondjacobson) [inline src/components/search/content/SearchItem.tsx:149]: really like how you structured this

**Review by raymondjacobson — APPROVED** [review]

File: src/components/search/TopBar.tsx (hunk 1 above)
Comment (jowlee) [reply src/components/search/TopBar.tsx:31]: This has a 1 px border bottom which I don't think is part of the height. - to match notifs

**Pull request merged**